client/my-sites/checkout/upsell-nudge/professional-email-upsell/style.scss

```
@@ -15,12 +15,80 @@
 
 .professional-email-upsell__header {
 	text-align: center;
+
+	.billing-interval-toggle {
+		display: flex;
+
+		.segmented-control {
+			background-color: #f2f2f2;
+			border: solid 1px var( --studio-gray-5 );
+			border-radius: 6px; /* stylelint-disable-line */
+			color: var( --color-text );
+
+			&.is-compact {
+				margin-bottom: 0;
+				max-width: unset;
+			}
+
+			.segmented-control__item {
+				border-radius: 6px; /* stylelint-disable-line */
+				border: 1px solid #f2f2f2;
```

Same again RE the colour - I think we should use a colour from our standard variables.
```suggestion
				border: 1px solid var( --studio-gray-0 );
```